specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-03-01/storagecache.json

```diff
@@ -1795,6 +1743,28 @@
         "statusDescription": {
           "description": "Describes explanation of state.",
           "type": "string"
+        },
+        "conditions": {
```

Comment:
Is this a readOnly property? If yes, please mark it as readonly.

Reply:
Updated in latest commit.